### PR TITLE
Extended README-docker for running the project locally.

### LINKED ISSUE #515: Extend README-docker.md
The current content of instructions for running the project stack under docker do not contain every steps that may be needed to a new comer in docker technology.

Hence there is need to put all the detailed instructions and descriptions that may help quickly build and run the project stack under docker.

### PR DESCRIPTION
**Pull request by sonlinux**
Fix #515 

Extended the instructions for running the project locally for the first time with useful step by step commands that worked for me.

### CODE REVIEW

File: deployment/README-docker.md [hunk 1]
```diff
@@ -85,6 +85,46 @@ e.g.
...
 ```
+### Run  project in development environment
+
+Firstly you need to be in the project **deployment** directory.
+
+```
+cd /path/to/feti/deployment/
+``` 
+
+Then after you may need to start the containers independently with the commands below, if your docker setup requires sudo privileges do include it.
```
Comment (cchristelis) [inline deployment/README-docker.md]: sudo is not needed if your user is in the dockers group. So lave out the sudo in the below commands.

File: deployment/README-docker.md [hunk 2]
```diff
@@ -85,6 +85,46 @@ e.g.
...
+sudo docker start feti_worker_1
+sudo docker start feti-dev-web
+```
+Then to start the docker development environment. 
+
+```
+make devweb
+```
+This should allow you to start the debug server within pycharm.
+But **easticsearch** engine need to be aware of the current data index to search from, with that in mind run the following commands if your data search is not working.
```
Comment (cchristelis) [inline deployment/README-docker.md]: elasticsearch <- typo

File: deployment/README-docker.md [hunk 3]
```diff
@@ -85,6 +85,46 @@ e.g.
...
+
+Assuming you have the **.dmp** database dump file, copy it under the following project directory
```
Comment (cchristelis) [inline deployment/README-docker.md]: ... as **latest.dmp**

**Review by cchristelis — COMMENTED** [review]
Thanks for updating this. Please update based on comments.

File: deployment/README-docker.md (hunk 1 above)
Comment (sonlinux) [reply deployment/README-docker.md]: Alright @cchristelis

File: deployment/README-docker.md (hunk 3 above)
Comment (sonlinux) [reply deployment/README-docker.md]: I have corrected that @cchristelis

File: deployment/README-docker.md (hunk 2 above)
Comment (sonlinux) [reply deployment/README-docker.md]: I have updated that.

**Comment by sonlinux** [comment]
Hi @cchristelis, could you take a look at this PR and comment for merging please.

**Pull request merged**